src/components/BrowseNavbar.tsx

```diff
@@ -0,0 +1,156 @@
+import { useEffect, useRef, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { signOut } from "firebase/auth";
+import { Bell, ChevronDown, Search } from "lucide-react";
+import logo from "../assets/logo.png";
+import { auth } from "../utils/firebase";
+
+const BrowseNavbar = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const dropdownRef = useRef<HTMLDivElement>(null);
+  const navigate = useNavigate();
+
+  const toggleDropdown = () => {
+    setIsOpen(!isOpen);
+  };
+
+  // Close the dropdown when clicking outside
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
+      if (
+        dropdownRef.current &&
+        !dropdownRef.current.contains(event.target as Node)
+      ) {
+        setIsOpen(false);
+      }
+    };
+
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, []);
+
+  function handleSignOut() {
+    signOut(auth)
+      .then(() => {
+        navigate("/login");
+      })
+      .catch((error) => {
+        console.log("error=", error);
+      });
+  }
+  return (
+    <div className="absolute w-screen z-50 text-white bg-gradient-to-b from-black">
+      <div className="flex justify-between">
+        <div className="flex">
+          <div>
+            <img
+              src={logo}
+              alt="logo"
+              height={100}
+              width={190}
+              className="ml-10"
+            />
+          </div>
+          <ul className="flex space-x-3 mt-6 ml-6 cursor-pointer">
```

Comment:
Can create a component that maps over an array of values (Home, Movies, etc) and create a ordered list. 
Keep this utility/shared component at parent level if it's being used elsewhere.

Reply:
Resolved